server/src/Api/Controllers/GameController.cs

```diff
@@ -22,4 +27,22 @@ public Task<int> CalculateScore(CalculateScoreCommand command)
     {
         return _mediator.Send(command);
     }
+    
+    [HttpPost(Routes.Game.Create)]
```

Comment:
Just `HttpPost` without specifying the endpoint name, because `HttpPost` is used to create new resources.
also for GetAllGames (HttpGet - Retrieve a representation of a resource) and for future HttpPut (Update resource), HttpDelete (Used to delete a resource)